.circleci/cimodel/data/windows_build_definitions.py

```diff
@@ -125,7 +125,7 @@ def FalsePred(_):
 def TruePred(_):
     return True
 
-_VC2019 = VcSpec(2019)
+_VC2019 = VcSpec(2019, ["14", "28", "29333"], hide_version=True)
```

Comment:
Shouldn't we stick with 14.27 (as latest 14.28 is broken and as far as and .29333 is no longer supported?

Reply:
OK, I'll try latest 16.7.13